code/modules/food_and_drinks/kitchen_machinery/icecream_vat.dm

```diff
@@ -155,13 +161,17 @@
 	else
 		var/obj/item/reagent_containers/food/snacks/examine_cone = new selected_cone
 		. += span_notice("<b>Alt Click</b> to dispense [examine_cone.name].")
+		//Delete item instance
+		qdel(examine_cone)
```

Comment:
```suggestion
		. += span_notice("<b>Alt Click</b> to dispense [selected_cone::name].")
```
i have no idea if this works it might need some edits

Reply:
This gave me errors, but a single : worked. I also used it with info_tab, but not for the scoop and cone part of the UI since I believe it won't work for getting their images.

Reply:
nononono that's not good at all, they are entirely different operators and ``:`` is something we should avoid

https://www.byond.com/docs/ref/#/operator/:
https://www.byond.com/docs/ref/#/operator/::